### PR TITLE
Update with new meeting time, effective 2016-03-21

### PR DESCRIPTION
**Pull request by bexelbie**
Proposed meeting time, decided in meeting on 2016-03-14.  Merges via lazy consensus on Thursday March 14, assuming it draws enough ACKs.

### CODE REVIEW

File: install-test [hunk 1]
```diff
@@ -0,0 +1,6 @@
+centos-libvirt
+fedora-libvirt
+centos-virtualbox
+fedora-virtualbox
+mac
+windows
```
Comment (navidshaikh) [inline install-test]: what is this `install-test` file?

Comment (brgnepal) [inline install-test]: @bexelbie This doesn't related to the PR. Is it extra?

**Comment by bexelbie** [comment]
@navidshaikh @budhrg repaired - please review and merge as appropriate.  Lazy consensus reached on the Mailing List

**Comment by brgnepal** [comment]
LGTM

**Comment by praveenkumar** [comment]
LGTM

**Comment by LalatenduMohanty** [comment]
@praveenkumar @bexelbie 1 week is very time for lazy consensus for a community meeting. Usually it takes  couple of weeks . 
Also -1 from me for the new suggested time as it is not favorable for users from USA and Canada as mentioned below.

Chicago Time : 5:00 AM
New York/Boston Time: 6:00AM
Canada Time: 6:00 AM

**Pull request merged**